external/dynamo/optimized/processor_multilru.py

```
@@ -0,0 +1,833 @@
+                                                     tokens_out,
+                                                     finish_reason)
+
+                    # Update core Prometheus metrics (fast atomic operations)
+                    self._metrics.request_latency_seconds.observe(latency_seconds)
+                    self._metrics.tokens_in_total.inc(tokens_in)
+                    self._metrics.tokens_out_total.inc(tokens_out)
+
+                    # Fire-and-forget KVE metric update (async, non-blocking)
+                    # This ensures KVE computation has ZERO impact on routing throughput
+                    if kve_data is not None:
+                        asyncio.create_task(self._update_kve_metrics_async(kve_data))
```

Ruff (0.14.14)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)